nl.json

```
@@ -583,7 +583,7 @@
   "thanks": "Dank u wel",
   "configuration_normal": "Normaal",
   "configuration_cargo_top": "Vracht op het dak",
-  "x_left_to_y": "nog {{x}} naar  {{y}}",
+  "x_left_to_y": "nog {{x}} naar  {{y}}",.
```

extra period should not be there
```suggestion
  "x_left_to_y": "nog {{x}} naar  {{y}}",
```